frontend-react/src/App.tsx

```diff
@@ -23,10 +24,14 @@ import { Upload } from "./pages/Upload";
 import { Suspense } from "react";
```

Comment:
I think we can nix this line now

Reply:
Ahh, dang! :joy: I always leave one import!